mmpose/models/heads/deeppose_regression_head.py

```diff
@@ -172,6 +172,9 @@ def decode(self, img_metas, output, **kwargs):
 
         preds, maxvals = keypoints_from_regression(output, c, s,
                                                    kwargs['img_size'])
+        if self.out_sigma:
+            maxvals = (1 - output[:, :, 2:].sigmoid())\
```

Comment:
Hi, does `output` has the attribute `sigmoid`? It is an np.ndarray according to the docstring.

Reply:
Thanks, it looks good to me now.

Reply:
I think we need a unit-test for this case. Especially for this bug.

Reply:
Agree. The unittest of `decode` interface is added